polaris/ocean/tests/inertial_gravity_wave/inertial_gravity_wave.cfg

```
@@ -0,0 +1,35 @@
+# config options for inertial gravity wave testcases
+[inertial_gravity_wave]
+
+# the size of the domain in km in the x and y directions
+lx = 10000
+f0 = 1e-4
```

```suggestion
coriolis_parameter = 1e-4
```
For consistency with other test cases